src/ai/backend/common/lock.py

```diff
@@ -240,13 +240,13 @@ async def __aenter__(self) -> None:
             thread_local=False,
             sleep=self._lock_acquire_pause,
         )
-        await self._lock.acquire()
+        await self._lock.__aenter__()
         if self._debug:
             log.debug("RedisLock.__aenter__(): lock acquired")
 
     async def __aexit__(self, *exc_info) -> Optional[bool]:
         assert self._lock is not None
-        await self._lock.release()
+        await self._lock.__aexit__(*exc_info)
```

Comment:
To make it more transparent, we should take and return the return value of inner `__aexit__()` call.

<img width="1029" alt="image" src="https://github.com/lablup/backend.ai/assets/555156/bcb01f5d-304f-444f-ac37-e9db482e0db8">